### DISCUSSION TITLE
Samsung, Dexcom, and recent Android patch

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2915] [General]
Please keep your comments as short as possible.

**Comment by Navid200** [comment]
1- Please verify the settings as Android update may have changed them: https://navid200.github.io/xDrip/docs/FAQ/AndroidUpdate.html

2- Please make sure you are using the latest xDrip Nightly.  
  
3- Please enable OB1 debugging: https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html  
3b- Restart the phone.  
  
4- If it is not paired, do what you usually do to pair.  Wait for it to work and fail.  
  
5- Look at the logs. You should see a lot more logs than usual.  
Only if you have waited for a while so that xDrip has operated with detail logs enabled, tap on "Upload logs" at the bottom.  
Add some relative narrative, like "gaps of x minutes between readings or whatever the problem is", and submit.  
  
6- Post in this thread and state you just uploaded including an identifying component in your upload so that the developer can communicate with you knowing who uploaded the logs.  
  
7- Please don't post here just because you have a problem.  Please allow this thread to be dedicated to the specific topic of the latest Samsung update impact.

**Comment by Andres-Ra** [comment]
Uploaded my log, device Samsung A33 5g (SM-A336B)

**Comment by jamorham** [comment]
Thank you for the logs. If you are affected by this problem please make the following settings changes to see if they help:
1) Turn off `Special pairing workaround`.
2) Turn off `Avoid scanning`

**Reply by Andres-Ra** [reply]
Turned both settings off and restarted transmitter. Connection came briefly back at some point and synced readings that were saved on transmitter, but now it has been 1h disconnected. Do you need new logs uploaded aswell after I change settings?

**Reply by Navid200** [reply]
Do you happen to also have a receiver?  If yes, please use it to keep your glucose under control until this is resolved.

**Reply by Andres-Ra** [reply]
Went back to oldschool and during the issue i'm using glucometer.

**Reply by Navid200** [reply]
Thanks.
We have limited resources and manpower.  This is important and we are working to resolve it.

**Comment by jamorham** [comment]
I know that people are reporting this issue after a system update but please if not tried already can anyone with the problem try rebooting their phone.

**Reply by Andres-Ra** [reply]
I've tried restarting multiple times, doesn't help.

**Comment by jamorham** [comment]
Please try the July 2nd Nightly (available now) to see if this helps? thanks

**Reply by Andres-Ra** [reply]
Unfortunately not..

NB! Can this be even xDrip problem if transmitter pairing is almost impossible? I mean I had to wait for ca 7 minutes with transmitter in my hand and touching the contacts, when the phone finally found the transmitter (just for pairing). After that it has been inactive for 3 hours. With prevous Samsung software version the pairing was basically instant (no contact touching was required).

**Comment by GDongelmans** [comment]
I uploaded a log from a samsung galaxy A53 5G xdrip version 35c1519-2023.07.02.

**Comment by parapenT1sta** [comment]
Can you check if changing Bluetooth, battery optimization from "Optimized" to "Non-Restrict" it helps?

![Imagem WhatsApp 2023-07-17 às 10 19 09](https://github.com/NightscoutFoundation/xDrip/assets/608650/f844ccd9-e353-40db-abd2-ad13dc0eda8a)

**Reply by jasom77** [reply]
I already tried that 2 weeks ago and there was no improvement.
It is clear that Samsung broke something in the update that affects to Bluetooth protocol used by cgm systems.

**Reply by parapenT1sta** [reply]
Thank you for sharing your experience.
